### PR TITLE
[HOPSWORKS-2403] Static website at examples.hopsworks.ai for the Notebooks

### PR DESCRIPTION
**Pull request by jimdowling**
https://logicalclocks.atlassian.net/browse/HOPSWORKS-2403

### CODE REVIEW

File: workspace/data/spam.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Dear sir, I am a Prince in a far kingdom you have not heard of.  I want to send you money via wire transfer so please ...
+Get Viagra real cheap!  Send money right away to ...
+Oh my gosh you can be really strong too with these drugs found in the rainforest. Get them cheap right now ...
+YOUR COMPUTER HAS BEEN INFECTED!  YOU MUST RESET YOUR PASSWORD.  Reply to this email with your password and SSN ...
+THIS IS NOT A SCAM!  Send money and get access to awesome stuff really cheap and never have to ...
```
Comment (SirOibaf) [inline workspace/data/spam.txt:5]: I doubt we need the `workspace/` directory committed in the repo. Especially this file

File: notebooks/featurestore/hsfs/external_table/external-table-delta-lake.ipynb [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.9"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```
Comment (SirOibaf) [inline notebooks/featurestore/hsfs/external_table/external-table-delta-lake.ipynb:77]: This notebook doesn't work. Let's try to not write garbage examples.

File: notebooks/featurestore/hsfs/serving/feature_engineering-online-fs.ipynb [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.9"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```
Comment (SirOibaf) [inline notebooks/featurestore/hsfs/serving/feature_engineering-online-fs.ipynb:88]: This notebook make no sense by itself. It's just a bunch of code with no explanation. Plus we already have a notebook for online serving - it's here: https://github.com/logicalclocks/hops-examples/blob/master/notebooks/featurestore/hsfs/feature_vector_model_serving.ipynb - Please get rid of this one.

File: Jenkinsfile_doc [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            agent {
+                node "local"
+            }
+
+            steps {
+                sh '''
+                  rm -rf /opt/examples/*
+                  rm -rf /opt/examples/.git
+                  rm -rf /opt/examples/.nojekyll
+		  cd /tmp
```
Comment (SirOibaf) [inline Jenkinsfile_doc:48]: You don't need to cd  in /tmp. Every Jenkins pipeline has a workspace to work with.

**Review by SirOibaf — CHANGES_REQUESTED** [review]

File: workspace/data/spam.txt (hunk 1 above)
Comment (jimdowling) [reply workspace/data/spam.txt:5]: Done

Comment (SirOibaf) [reply workspace/data/spam.txt:5]: You didn't push anything.

Comment (jimdowling) [reply workspace/data/spam.txt]: Pushed

File: notebooks/featurestore/hsfs/external_table/external-table-delta-lake.ipynb (hunk 2 above)
Comment (jimdowling) [reply notebooks/featurestore/hsfs/external_table/external-table-delta-lake.ipynb]: Does a new feature exist if there's no example of it.

Comment (jimdowling) [reply notebooks/featurestore/hsfs/external_table/external-table-delta-lake.ipynb]: Removed it.

File: notebooks/featurestore/hsfs/serving/feature_engineering-online-fs.ipynb (hunk 3 above)
Comment (jimdowling) [reply notebooks/featurestore/hsfs/serving/feature_engineering-online-fs.ipynb:88]: The examples site should not just contain long-form notebooks. It should be a quick page to find answers to problems you have. How do i write a csv training dataset in 1 file? 
They are HOW-TO docs. We have a mix of tutorials and HOW-TO. You cannot say they should be just tutorials - that would defeat the purpose of them.

We are missing a notebook for "How do i ingest into the online FS". That is a different question from "How do i do online feature serving". We need to answer more questions. If the code is self explanatory, we should not need reems of (tutorial) text around it. We shouldn't have to show online feature vector serving if all I want to see is how to ingest into the online FS.

File: Jenkinsfile_doc (hunk 4 above)
Comment (jimdowling) [reply Jenkinsfile_doc]: done.

**Review by moritzmeister — APPROVED** [review]

**Review by SirOibaf — APPROVED** [review]

**Pull request merged**